src/x11/x.c

```diff
@@ -703,6 +703,14 @@ static void x_win_move(int width, int height)
                 y = scr->dim.y + xctx.geometry.y;
         }
 
+        if (xctx.geometry.center && xctx.geometry.x == 0) {
+                x = (scr->dim.x + (scr->dim.w - width) / 2);
```

Comment:
Why is the assignment enclosed in parenthesis? They are not needed and add nothing to readability.